argoverse/data_loading/argoverse_stereo_loader.py

```diff
@@ -0,0 +1,459 @@
+# <Copyright 2021, Argo AI, LLC. Released under the MIT license.>
+
+import glob
+import logging
+import os
+from pathlib import Path
+from typing import Dict, Iterator, List, Optional, Union
+
+import cv2
+import numpy as np
+
+from argoverse.utils.calibration import Calibration, load_stereo_calib
+from argoverse.utils.camera_stats import RECTIFIED_STEREO_CAMERA_LIST
+
+logger = logging.getLogger(__name__)
+
+STEREO_FRONT_LEFT_RECT = RECTIFIED_STEREO_CAMERA_LIST[0]
+STEREO_FRONT_RIGHT_RECT = RECTIFIED_STEREO_CAMERA_LIST[1]
+
+STEREO_DISPARITY_LIST = ["stereo_front_left_rect_disparity", "stereo_front_left_rect_objects_disparity"]
+
+DISPARITY_NORMALIZATION: float = 2.0 ** 8  # 256.0
+
+
+class ArgoverseStereoLoader:
+    def __init__(self, root_dir: str, split_name: str) -> None:
+        # initialize class member
+        self.RECTIFIED_STEREO_CAMERA_LIST = RECTIFIED_STEREO_CAMERA_LIST
+        self._log_list: Optional[List[str]] = None
+        self._image_list: Optional[Dict[str, Dict[str, List[str]]]] = None
```

Comment:
inline comments here might help, e.g.
```python
# for each log, provides image paths for each rectified stereo camera
self._image_list: Optional[Dict[str, Dict[str, List[str]]]] = None
```
It does seem like we are using a dictionary as a class, whereas NamedTuple is preferred for that. But at least inline comments would be super helpful.